services/apps/integration_run_worker/package.json

```diff
@@ -20,14 +20,16 @@
     "script:continue-run": "SERVICE=script TS_NODE_TRANSPILE_ONLY=true node -r tsconfig-paths/register -r ts-node/register src/bin/continue-run.ts"
   },
   "dependencies": {
+    "@crowd/alerting": "file:../../libs/alerting",
     "@crowd/common": "file:../../libs/common",
     "@crowd/database": "file:../../libs/database",
     "@crowd/integrations": "file:../../libs/integrations",
     "@crowd/logging": "file:../../libs/logging",
     "@crowd/redis": "file:../../libs/redis",
     "@crowd/sqs": "file:../../libs/sqs",
+    "@crowd/tracing": "file:../../libs/tracing",
     "@crowd/types": "file:../../libs/types",
-    "@crowd/alerting": "file:../../libs/alerting",
+    "@opentelemetry/api": "~1.6.0",
```

Comment:
You sure you need this here?

Reply:
Yes, to have access to the `Tracer` interface and `SpanStatusCode` enum.

Reply:
Then let's do it like here:
https://github.com/CrowdDotDev/crowd.dev/blob/ae65bf25f23e61ca1e6b3f44872080da67f7be30/services/libs/logging/src/types.ts#L1-L3

Otherwise we're just going to add this dependency everywhere